ui/src/components/Swap.jsx

```diff
@@ -1,11 +1,11 @@
 import { React, useState, useEffect } from 'react';
 import { E } from '@endo/captp';
 import {
-  divideBy,
+  floorDivideBy,
```

Comment:
These changes are unrelated, but necessary to fix a compilation error, I figured it would be easy enough to include them in this fix